POLICY.md

```diff
@@ -0,0 +1,55 @@
+# Compatibility & deprecation
+
+Until TaskChampion reaches [v1.0.0](https://github.com/taskchampion/taskchampion/milestone/7), nothing is set in stone. That being said, we aim for the following:
+
+1. Major versions represent significant change and may be incompatible with previous major release.
+2. Minor versions are always backwards compatible and might add some new functionality.
+3. Patch versions should not introduce any new functionality and do what name implies &mdash; fix bugs.
+
+As there are no major releases yet, we do not support any older versions. Users are encouraged to use the latest release.
+
+## ABI policy
+
+1. We target stable `rustc`.
+2. TaskChampion will never upgrade any backend to a non-compatible version without explicit user's request.
+
+## API policy
```

Comment:
What constitutes an API in this sense?  CLI?  Rust API?  Replica/server HTTP API?

Reply:
CLI and HTTP, Rust would be ABI I believe. Re-shuffled text to make it clearer.